prestodb/centos6-oj8-openldap/Dockerfile

```diff
@@ -73,6 +73,7 @@ RUN keytool -import -alias caroot -storepass testldap -keystore /etc/openldap/ce
     -file /etc/openldap/certs/caroot.cer
 
 # Install supervisor.d
-RUN yum install -y epel-release && yum install -y python-pip && pip install supervisor && yum -y clean all && rm -rf /tmp/* /var/tmp/*
+RUN yum install -y setuptools epel-release && yum install -y python-pip && pip install --upgrade pip && pip install --upgrade setuptools && pip install supervisor && \
```

Comment:
[nit] make && to end the line. That way it will be easier to read.

Reply:
that was following your last changes, but sure, will change this :)